src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -3033,15 +3033,15 @@ TEST_F(EnsembleConfigurationValidationWithDemultiplexer, MultipleBatchInCustomNo
         {NodeKind::ENTRY, ENTRY_NODE_NAME, "", std::nullopt, {{pipelineInputName, pipelineInputName}}},
         {NodeKind::CUSTOM, "custom_node_1", "", std::nullopt, {{"1", "out_OutputNumbers_1"}, {"2", "out_OutputNumbers_2"}}, demultiplyCount, {}, mockedLibrary,
             parameters_t{
-                {"in_InputNumbers", "3,3,10;FP32"},  // 1,3,10 is correct
+                {"in_InputNumbers", "3,3,10;FP32"},
                 {"out_OutputNumbers_1", "9,1,700;I32"},
```

Comment:
Why do we need to create new node to pass shapes via parameters if we could use this mocked library?

Reply:
created Library mock and used it in a test instead of creating new node